pkg/kp/kv.go

```diff
@@ -9,10 +9,17 @@ import (
 	"time"
 
 	"github.com/hashicorp/consul/api"
+	"gopkg.in/alecthomas/kingpin.v1"
+
 	"github.com/square/p2/pkg/pods"
 	"github.com/square/p2/pkg/util"
 )
 
+var showConsulErrors = kingpin.Flag(
```

Comment:
ah interesting, haven't thought of putting a kingpin flag in a pkg before (as opposed to something in bin), but it seems to work for what we want here, eh